src/main/webapp/go.js

```diff
@@ -231,7 +285,7 @@ function toggleProceedButton(hide) {
   if (hide) {
     proceedButton.classList.add(INVISIBLE_CLASS);
   } else {
-    if (destIndex === huntArr.length - 1) {
+    if (hunt.getDestIndex() === hunt.getNumItems() - 1) {
```

Comment:
Could create a helper in ScavengerHuntManager to check this, i.e.

if (hunt.isAtLastStop())